src/com/egs/arsensargsyan/task2/ReadFromFileIntoArrayList.java

```diff
@@ -0,0 +1,110 @@
+package task2;
+
+import java.io.*;
+import java.util.ArrayList;
+import java.util.Scanner;
+
+public class ReadFromFileIntoArrayList {
+
+    private static ArrayList<User> users = new ArrayList<>();
+    private static final File FILE_PATH = new File("arsen-sargsyan-resources/fileArrayList");
+
+    public static void main(String[] args) {
+       new ReadFromFileIntoArrayList().go();
+    }
+
+    public void go() {
+        try {
+            readFile();
+        } catch (Exception e) {
+            System.out.println("Error during read");
+        }
+        try {
+            writeFile();
+        } catch (IOException e) {
+            System.out.println("Error during  write  or console ");
+        }
+    }
+
+    private static void readFile() throws IOException, ClassNotFoundException {
+        try (BufferedReader bRid = new BufferedReader(new FileReader(FILE_PATH))) {
+            String str;
+            while ((str = bRid.readLine()) != null) {
+                String[] userNameAndAge = str.split(" ");
+                users.add(new User(userNameAndAge[0], Integer.parseInt(userNameAndAge[1])));
+            }
+        }
+    }
+
+    private static void writeFile() throws IOException {
+        System.out.println("\nselect <command(ADD,REMOVE,LIST or EXIT)\n" +
+                "for <add> select :ADD space age: \n" +
+                "for <remove> select :REMOVE space Index(n): \n" +
+                "for <Print> select only: LIST:\n" +
+                "for <exit and Save> select only :EXIT:\n" +
+                "for <support> select only :HELP:\n" +
+                "for <clear> select only :CLEAR:");
+        Scanner scanner = new Scanner(System.in);
+        String line;
+        String[] array = {""};
+        while (!array[0].equals("EXIT")) {
+            System.out.println("\nInput command");
+            line = scanner.nextLine();
+            array = line.split(" ");
+            switch (array[0].toUpperCase()) {
+                case "ADD": {
+                    users.add(new User(array[1], Integer.parseInt(array[2])));
+                    System.out.println("user added");
+                    break;
+                }
+                case "REMOVE": {
+                    if (array.length < Integer.parseInt(array[1])) {
```

Comment:
you are going to break the program if there is no such user?
wouldn't be better just let user know that there is no such user and he can try his chance once again?